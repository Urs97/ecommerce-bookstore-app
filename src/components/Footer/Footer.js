import React from 'react';
import './Footer.scss';
import { Link } from 'react-router-dom';

function Footer() {
    return (
        <section className="footer">
            <section className="footer-nav">
                <Link to="/">Home</Link>
                <Link to="/shop">Shop</Link>
                <Link to="/about">About Us</Link>
                <Link to="/about">TOS</Link>
                <Link to="/about">Contact Us</Link>
            </section>
            <p>Bookz Â© 2020 All rights reserved</p>
        </section>
    )
}

export default Footer;