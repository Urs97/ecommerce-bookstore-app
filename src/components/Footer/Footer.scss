@import '../../styles/shared.scss';

.footer {
    color: #FFFFFF;
    background-color: $grey;
    padding: 5em 0;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    
    & a {
        text-decoration: none;
        color: #FFFFFF;
        padding-bottom: 0.2em;
        outline: none;
    }

    & a:hover {
        color: #000000;
    }

    & a:focus {
        outline: 0.2em solid #FFFFFF;
    }

    & a:not(.focus-visible) {
        outline: none;
    }

    & a:not(:first-child) {
        margin-left: 1.5em;
    }

    p {
        margin-top: 1.5em;
    }
}

@media (max-width: 1260px) {
    .footer {
        font-size: 1.2em;
    }
}