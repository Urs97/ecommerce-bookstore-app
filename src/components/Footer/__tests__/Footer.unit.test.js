import React from "react";
import { shallow } from "enzyme";
import Footer from "../Footer";

describe("Footer", () => {
  test("renders Footer", () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
