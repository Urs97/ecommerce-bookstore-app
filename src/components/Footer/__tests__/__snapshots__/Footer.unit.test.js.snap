// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders Footer 1`] = `
<footer
  className="footer"
>
  <section
    className="footer-nav"
  >
    <Link
      to="/"
    >
      Home
    </Link>
    <Link
      to="/store"
    >
      Store
    </Link>
    <Link
      to="/cart"
    >
      Cart
    </Link>
    <Link
      to="/about"
    >
      About Us
    </Link>
  </section>
  <p>
    Bookz Â© 2020 All rights reserved
  </p>
</footer>
`;
